cpp/data/sql_updates/ub_tools.24

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE metadata_presence_tracer ADD COLUMN marc_subfield_code CHAR(1) NOT NULL AFTER marc_field_tag;
+ALTER TABLE metadata_presence_tracer CHANGE COLUMN zeder_journal_id journal_id INT(11) UNSIGNED;
```

Comment:
What about the change to the metadata_presence_tracer_zeder_journal_id CONSTRAINT? Will this be changed implicitly as well?

Reply:
I would hope so but am not 100% certain.

Reply:
Taken care of.